cmd/xchains-api/main.go

```diff
@@ -22,6 +22,7 @@ func init() {
 	if err := godotenv.Load(); err != nil {
 		log.Debug().Msg("failed to load .env file")
 	}
+	types.InitCovenantParams()
```

Comment:
Change to Lazy pattern GetLazyValue() { 
if value == nil {value = SomeValue}
return value
}